src/common/ceph_crypto.cc

```diff
@@ -27,7 +27,7 @@ void ceph::crypto::init(CephContext *cct)
 {
 }
 
-void ceph::crypto::shutdown(bool shared)
+void ceph::crypto::shutdown(bool)
```

Comment:
ic. it does help to keep the history of review and amendments. but we rely on our contributors to do the squash, before merging. but if the change can be split into a series of logically separated changes, it would be ideal to keep them as they are in our commit history.

if your change is non trivial, and you are addressing the reviewer's comment in your change, it's advisable to have the fix addressing the comments on top of existing change, and squash it into the related commit, once the reviewer acks your fix.